src/containers/RandomCocktails.js

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import axios from "axios";
+
+class RandomCocktail extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      dataRecipe: {},
+    };
+    this.getCocktail = this.getCocktail.bind(this);
+  }
+
+  componentDidMount() {
+    this.getCocktail();
+  }
+
+  getCocktail() {
+    axios
```

Comment:
please extract axios logic in other file

Reply:
in progress for the next sprint